src/features/curriculum/curriculum-route.ts

```diff
@@ -75,11 +76,14 @@ router.post('/topic',
 router.put('/unit/:id',
     authenticationMiddleware,
     validate(updateCurriculumUnitValidation),
-    asyncHandler(async (req: Request, res: Response, next: NextFunction) => {
+    // This is a typescript workaround since this comes up as extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I want to be yelled at since I there are very few cases I actually want to use this
In this case it's because express generic types are too restrictive so the type doesn't work, however doing it with the workaround behaves as expected